lib/shrinkwrapper.js

```diff
@@ -88,35 +90,36 @@ function fixWinEPERMSync (p, err) {
   }
 }
 
-function rmkidsSync (p) {
+function rmkidsSync(p) {
   fs.readdirSync(p).forEach(function (f) {
     rimrafSync(path.join(p, f));
   });
   fs.rmdirSync(p);
 }
 
 
-function npm (cmd) {
+function npm(cmd) {
   var result = spawn(npmCmd, cmd.split(' '), { cwd: BASEDIR, stdio: 'inherit' });
   if (result.status !== 0) {
     process.exit(result.status);
   }
   return result;
 }
 
-function gitDep (rep) {
+function gitDep(rep) {
   return /^git/.test(rep);
 }
 
-function replacer (key, val) {
+function replacer(key, val) {
+  /*eslint no-invalid-this: 0 */
   if (!this.version) {
     return val;
   }
-  if (key === "from" && !gitDep(this.resolved)) {
-    return undefined;
+  if (key === 'from' && !gitDep(this.resolved)) {
+    return void (0);
```

Comment:
Stupid linting rule just adds noise; might be better to just disable it (the one that forces you to return a value) at least for this block.